src/app/creator-profile-page/creator-profile-nfts/creator-profile-nfts.component.ts

```diff
@@ -122,12 +122,14 @@ export class CreatorProfileNftsComponent implements OnInit {
   }
 
   getNFTs(isForSale: boolean | null = null): Subscription {
+    const isPending = isForSale === null;
```

Comment:
`isForSale` is only null when the activeTab is `My Gallery`. This would make us fetch only pending NFTs on the `My Gallery` view which I don't believe is desired.  Can you clarify when we're supposed to fetch pending NFTs or what you're trying to accomplish with this line?

Reply:
> `isForSale` is only null when the activeTab is `My Gallery`. This would make us fetch only pending NFTs on the `My Gallery` view which I don't believe is desired. Can you clarify when we're supposed to fetch pending NFTs or what you're trying to accomplish with this line?

Apologies, you're right; this is doing exactly opposite of what I intended. 🤦‍♂️  Apologies. Just trying to filter pending NFTs from the Gallery.

I'm unable to build backend locally so I wasn't able to test this. Will push a fix when I'm able to get back to the code.

Reply:
You should be able to build it locally now - let me know if you're still having issues.